### ISSUE TITLE
Kmeans requests

### ISSUE THREAD

**Issue by domfournier** [MiraGeoscience/geoapps#34]
From @orerocks

- Add different normalizations: sklearn.preprocessing.StandardScaler would be a good start, and probably PowerTransformer for log-normal data. 
- Add option for k_init: we want to run lots of iterations for simple problems and few for large ones.

**Comment by domfournier** [comment]
GEOPY-216

**Comment by domfournier** [comment]
Duplicate of #76
